### ISSUE TITLE
Fix post regression analysis

### ISSUE THREAD

**Issue by diegoinacio** [diegoinacio/diegoinacio.github.io#10]
Logistic Regression is missing loss function
Normalize gradient descent
